classes/Device.class.php

```diff
@@ -2192,7 +2195,13 @@ static function UpdateSensors($CabinetID=null){
 			// Strip out everything but numbers
```

Comment:
Unfortunatly the `$temp*=$t->TempMultiplier;` (and `$humidity`) line will otherwise just remove the sub-decimal (?) values. 

On the other hand, I just realized, that the issue might be another code, as an snmpget on the cli will return "properly" formatted numbers. (for example "50.2")